File: MODUL 5/SOAL 5/PRAK405-2210817210026-Zulfa Auliya Akbar.c
#include <stdio.h>

int main(){

    int n,k,i,j,nilai1,nilai2,total;
    nilai1 = nilai2 = total = 0;
    scanf("%d %d", &n, &k);
    for(i=1;i<=n;i++){
        for(j=i;j>=1;j--){
            printf("\(%d * %d\) ",j,k);
            if (j>1){
                printf(" + ");
            }
        }
        nilai1 = (i*k);
        nilai2 += nilai1;
        printf (" = %d",nilai2);
        total+= nilai2;
        printf("\n");
    }
    printf("%d",total);
}

File: MODUL 5/SOAL 5/PRAK405-2210817210026-Zulfa Auliya Akbar.py
n,k=map(int,input().split())
nilai1 = nilai2 = total = 0

for i in range(1 ,n+1):
    j = i
    while(j>=1):
        print("(%d * %d)"%(j,k),end=' ')
        if j>1:
            print("+",end=' ')
        j-=1

    nilai1=i*k
    nilai2+=nilai1
    print("= %d"%(nilai2),end=' ')
    print(" ")
    total+=nilai2

print(total)